src/components/EnvoyFleets/EnvoyFleetInfo/StaticRoutes/StaticRoutes.tsx

```diff
@@ -10,25 +14,49 @@ const columns = [
 ];
 
 const StaticRoutes: React.FC = () => {
-  const staticRoutes = useAppSelector(state => state.main.selectedEnvoyFleet?.staticRoutes);
+  const dispatch = useAppDispatch();
+  const envoyFleetStaticRoutes = useAppSelector(state => state.main.selectedEnvoyFleet?.staticRoutes);
+  const staticRoutes = useAppSelector(state => state.main.staticRoutes);
+
+  const navigate = useNavigate();
 
   const dataSource = useMemo(() => {
-    if (!staticRoutes?.length) {
+    if (!envoyFleetStaticRoutes?.length) {
       return [];
     }
 
-    return staticRoutes.map(staticRoute => ({
-      key: `${staticRoute.namespace}-${staticRoute.name}`,
-      name: staticRoute.name,
-      namespace: staticRoute.namespace,
+    return envoyFleetStaticRoutes.map(envoyFleetStaticRoute => ({
+      key: `${envoyFleetStaticRoute.namespace}-${envoyFleetStaticRoute.name}`,
```

Comment:
I would destructure this `envoyFleeStaticRoute` in the function block.
Would look like this: 
`envoyFleetStaticRoute => {
    const {name, namespace} = envoyFleetStaticRoute;
    return {...}
}`